nin/dasBoot/ShaderPassNode.js

```diff
@@ -0,0 +1,36 @@
+(function(NIN) {
+  'use strict';
+
+  class ShaderPassNode extends NIN.Node {
+    constructor(id, options) {
+      super(id, {
+        inputs: options.inputs,
+        outputs: {
+          A: new NIN.TextureOutput()
+        }
+      });
+      this.camera = new THREE.OrthographicCamera(-1, 1, 1, -1, 0, 1); 
+      this.scene = new THREE.Scene();
+      this.quad = new THREE.Mesh(new THREE.PlaneBufferGeometry(2, 2),
+                                 new THREE.ShaderMaterial(options.shader));
+      this.shader = this.quad.material;
+      this.scene.add(this.quad);
+      this.renderTarget = new THREE.WebGLRenderTarget(640, 360, {
```

Comment:
No, for reference:

![image](https://cloud.githubusercontent.com/assets/578029/17004446/44ad49d6-4ed5-11e6-8b42-886472084c52.png)